src/pages/static/js/tabs.js

```diff
@@ -0,0 +1,84 @@
+class TabsBox extends HTMLElement {
+    constructor() {
+      super();
+      this.attachShadow({ mode: 'open' });
+      this.render();
+      this.loadTabs();
+    }
+  
+    async loadTabs() {
```

Comment:
We shouldn't be loading the active tabs of a particular window in peersky://tabs. We want to list tab groups here, the same way we do on the title bar.

See my comment: https://github.com/p2plabsxyz/peersky-browser/pull/48#discussion_r2168004733